packages/site-kit/src/lib/markdown/utils.js

```diff
@@ -212,10 +212,8 @@ const default_renderer = {
  * @param {string} markdown
  * @param {Partial<import('marked').Renderer>} renderer
  */
-export function transform(markdown, renderer = {}) {
-	marked.use({
-		mangle: false,
-		headerIds: false,
+export async function transform(markdown, renderer = {}) {
+	const marked = new Marked({
 		renderer: {
 			// we have to jump through these hoops because of marked's API design choices —
 			// options are global, and merged in confusing ways. You can't do e.g.
```

Comment:
This comment can probably be removed now

Reply:
Done